src/index.ts

```diff
@@ -890,7 +923,11 @@ export type HoldToken = {
 }
 
 export type StyleOverride = {
-    font?: 'Roboto' | 'Montserrat' | 'WorkSans' | 'NotoSansHK' | 'Lato' | 'NunitoSans'
+    font?: 'Inter' | 'Roboto' | 'Montserrat' | 'WorkSans' | 'NotoSansHK' | 'Lato' | 'NunitoSans'
+    cornerRadius?: 'round' | 'square'
+    buttonShape?: 'round' | 'square'
+
+    // Deprecated:
```

Comment:
I think we need separate `// @deprecated` comments for all deprecated properties. If not, IDEs won't properly show them as deprecated. At least IntellIJ doesn't.

Reply:
Ah right!